src/densities.jl

```diff
@@ -79,3 +79,23 @@ be one coefficient matrix per k-Point.
     count = sum(length(basis.ksymops[ik]) for ik in 1:length(basis.kpoints))
     from_fourier(basis, sum(ρaccus) / count)
 end
+
+"""
+Computes the *probability* (not charge) current, ∑ fn Im(ϕn* ∇ϕn)
```

Comment:
Why is this in this file?

Reply:
It's pretty similar to a density, it's a "current density"... I don't want to create a `current.jl` file just for this.

Reply:
I think more files are better. In my head it is definitely a different concept.

Reply:
It kind of belongs with DOSes as "density-like things we might want to compute". I can put it there in postprocess